assets/src/scripts/components/hydrograph/index.js

```diff
@@ -87,8 +87,10 @@ class Hydrograph {
         select('#ts-current')
             .attr('d', this.currentLine(this._tsData.current));
 
-        //redraw the median stats scatter plot
-        this.svg.selectAll('#median-point').call(this._plotMedianPoints, this.scale, this._medianStatsData);
```

Comment:
Doing this was the page to freeze up after a few clicks of the "compare" checkbox and wasn't successfully redrawing the median statistic points when the y-axis updates.